packages/front-end/pages/settings/index.tsx

```diff
@@ -269,7 +272,12 @@ const GeneralSettingsPage = (): React.ReactElement => {
 
   const { metricDefaults } = useOrganizationMetricDefaults();
 
+  const { data, mutate } = useApi<{
+    checklist: ExperimentLaunchChecklistInterface;
+  }>("/experiments/launch-checklist");
```

Comment:
Why load this on the settings page?  Would be better to load within the modal.  Avoids extra unnecessary API requests just from viewing this page.